result of this hack is that POSIX_LCTime will generate correct alt_digits.
This is not final solution, we need to find better way how to detect that  alt_digits are necessary.

--- cldr_core.orig/common/main/ar.xml
+++ cldr_core/common/main/ar.xml
@@ -5278,6 +5278,7 @@
 			<group>٬</group>
 			<list>؛</list>
 			<percentSign>٪</percentSign>
+			<nativeZeroDigit>٠</nativeZeroDigit>
 			<plusSign>‏+</plusSign> <!-- includes RLM before sign -->
 			<minusSign>‏-</minusSign> <!-- includes RLM before sign (002D) -->
 			<exponential>اس</exponential>
@@ -5292,6 +5293,7 @@
 			<group>,</group>
 			<list>;</list>
 			<percentSign>٪</percentSign>
+			<nativeZeroDigit>٠</nativeZeroDigit>
 			<plusSign>‎+</plusSign> <!-- includes LRM before sign -->
 			<minusSign>‎-</minusSign> <!-- includes LRM before sign (002D) -->
 			<exponential>E</exponential>
